notes/returned-mail-received.vbs

```diff
@@ -191,70 +227,64 @@ IF mailing_address_confirmed = "YES" or residential_address_confirmed = "YES" TH
-		  CheckBox 100, 15, 90, 10, "Shelter Form (DHS-2952)", SVF_checkbox
-		  CheckBox 10, 25, 80, 10, "Verification Request", verif_request_checkbox
-		  CheckBox 100, 25, 100, 10, "Change Report (DHS-2402)", CRF_checkbox
-		  EditBox 40, 55, 155, 15, new_addr_line_one
-		  EditBox 40, 75, 155, 15, new_addr_line_two
-		  EditBox 40, 95, 155, 15, new_addr_city
-		  EditBox 40, 115, 20, 15, new_addr_state
-		  EditBox 155, 115, 40, 15, new_addr_zip
-		  DropListBox 130, 140, 65, 15, "Select:"+chr(9)+county_list, county_code
-		  DropListBox 140, 165, 55, 15, "Select:"+chr(9)+"YES"+chr(9)+"NO"+chr(9)+"N/A", mets_addr_correspondence
-		  EditBox 140, 180, 55, 15, mets_case_number
-		  EditBox 55, 200, 140, 15, other_notes
-		  ButtonGroup ButtonPressed
-		    OkButton 85, 220, 50, 15
-		    CancelButton 145, 220, 50, 15
-		  GroupBox 5, 5, 195, 35, "Verification Request Includes:"
-		  GroupBox 5, 40, 195, 120, "Forwarding Address:"
-		  Text 10, 60, 25, 10, "Street:"
-		  Text 20, 100, 15, 10, "City:"
-		  Text 20, 120, 20, 10, "State:"
-		  Text 120, 120, 30, 10, "Zip code:"
-		  Text 100, 145, 25, 10, "County:"
-		  Text 10, 170, 95, 10, "METS correspondence sent:"
-		  Text 10, 185, 70, 10, "METS case number:"
-		  Text 10, 205, 40, 10, "Other notes:"
-		EndDialog
+    	BeginDialog Dialog1, 0, 0, 206, 215, "RETURNED MAIL PROCESSING"
+    	  CheckBox 10, 15, 75, 10, "Returned Mail/Other", return_mail_checkbox
+    	  CheckBox 100, 15, 90, 10, "Shelter Form (DHS-2952)", SVF_checkbox
+    	  CheckBox 10, 25, 80, 10, "Verification Request", verif_request_checkbox
+    	  CheckBox 100, 25, 100, 10, "Change Report (DHS-2402)", CRF_checkbox
+    	  EditBox 40, 55, 155, 15, new_addr_line_one
+    	  EditBox 40, 75, 155, 15, new_addr_line_two
+    	  EditBox 40, 95, 155, 15, new_addr_city
+    	  EditBox 40, 115, 20, 15, new_addr_state
+    	  EditBox 155, 115, 40, 15, new_addr_zip
+    	  DropListBox 145, 140, 55, 15, "Select:"+chr(9)+"YES"+chr(9)+"NO"+chr(9)+"N/A", mets_addr_correspondence
+    	  EditBox 145, 155, 55, 15, mets_case_number
+    	  EditBox 60, 175, 140, 15, other_notes
+    	  ButtonGroup ButtonPressed
+    	    OkButton 95, 195, 50, 15
+    	    CancelButton 150, 195, 50, 15
+    	  GroupBox 5, 5, 195, 35, "Verification Request Includes:"
+    	  GroupBox 5, 40, 195, 95, "Forwarding Address:"
+    	  Text 10, 60, 25, 10, "Street:"
+    	  Text 10, 100, 15, 10, "City:"
+    	  Text 10, 120, 20, 10, "State:"
+    	  Text 120, 120, 30, 10, "Zip code:"
+    	  Text 10, 145, 95, 10, "METS correspondence sent:"
+    	  Text 10, 160, 70, 10, "METS case number:"
+    	  Text 10, 180, 40, 10, "Other notes:"
+    	EndDialog
 
     	DO
     		DO
     			err_msg = ""
     			DIALOG Dialog1
     			cancel_without_confirmation
-
-				new_addr_line_one = trim(new_addr_line_one)
-				new_addr_line_two = trim(new_addr_line_two)
-				new_addr_city = trim(new_addr_city)
+				new_addr_line_one = trim(UCASE(new_addr_line_one))
```

Comment:
Was this needed for the script to work?

Reply:
It is not but it creates consistency in the case note